apps/indexer/src/api/repositories/drizzle.ts

```diff
@@ -156,6 +175,136 @@ export class DrizzleRepository {
     return db.$count(proposalsOnchain);
   }
 
+  async getProposalNonVoters(
+    proposalId: string,
+    skip: number,
+    limit: number,
+    orderDirection: "asc" | "desc",
```

Comment:
Maybe we could put these literals in a constant/enum at some point, as they are re-declared a lot around the project